fontbakery-check-ttf.py

```diff
@@ -105,6 +105,33 @@
 NAMEID_LIGHT_BACKGROUND_PALETTE = 23
 NAMEID_DARK_BACKGROUD_PALETTE = 24
 
+NAMEID_STR = {
```

Comment:
Both key and value in this dict may look the same for us, coders, glancing at a text editor screen. But they are actually different beasts. The keys are numbers while the values are text strings. At runtime the python interpreter will lookup this dict with a name entry nameID numeric value and will print the appropriate text output in error messages.

For instance, this is useful here:
https://github.com/felipesanches/fontbakery/blob/e4014f662fbd6bf8c906c77c7cd3a8f64754006e/fontbakery-check-ttf.py#L802-L804